### ISSUE TITLE
Breaking change might have duplicated headers

### ISSUE THREAD

**Issue by ctron** [convco/convco#222]
**Describe the bug**

Having a breaking change in different variants `BREAKING CHANGE` and `BREAKING-CHANGE`, will actually create two different sections.

**To Reproduce**
1. Clone https://github.com/patternfly-yew/patternfly-yew/
2. Run `convco ...`
4. See error ...

**Expected behavior**

I would expect breaking changes to get collapsed into a single group.

**System (please complete the following information):**
 - OS: `fedora`, `ubuntu`, …
 - Version: 39, 22.04, …

**Additional context**

**Comment by hdevalke** [comment]
Footers are stored as :

```rust
pub(crate) struct Footer {
    pub(crate) key: String,
    pub(crate) value: String,
}
```

Easiest option would be to change the footer to 


```rust
pub(crate) struct Footer {
    pub(crate) key: FooterKey,
    pub(crate) value: String,
}

enum FooterKey {
    BreakingChange,
    Key(String),
}
```

However, we would have to decide to stringify it to either `BREAKING CHANGE` or `BREAKING-CHANGE`.

@ctron do you have any preference? If not, I would go for `BREAKING CHANGE`

**Comment by hdevalke** [comment]
Change also implemented in #223

**Comment by ctron** [comment]
> @ctron do you have any preference? If not, I would go for `BREAKING CHANGE`

:+1: Makes sense to me.
